slot/src/main.ts

```diff
@@ -0,0 +1,9 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+import { Transport } from '@nestjs/microservices'
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  await app.listen(process.env.PORT || 3000);
```

Comment:
no process env directly it should be read from config, it would be better to have one config in some common place and use it in entire app